app/views/photos/show.html.haml

```diff
@@ -0,0 +1,6 @@
+%p= @photo.name
+= image_tag @photo.image.url(:thumb)
+%p= @photo.description
+= link_to "Delete post", photo_path(@photo), method: :delete
+
+%a(href='/photos') Back to feed
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))